app/controllers/concerns/verify_sp_attributes_concern.rb

```diff
@@ -36,6 +40,12 @@ def clear_verify_attributes_sessions
     user_session[:verify_shared_attributes] = false
   end
 
+  def consent_has_expired?
+    return true unless sp_session_identity
+    last_estimated_consent = sp_session_identity.last_consented_at || sp_session_identity.created_at
```

Comment:
Would love some feedback on this specifically:

1. Do we think it's ok to use `created_at` as a proxy for the last time we know for sure somebody consented on the handoff page?

  - I can think of a case where the `identities` row is created, and then abandoned, and then viewed later, so the timestamp may be older than expected

2. Do we want to take the time to run a backfill to set `last_consented_at = created_at` on the entire table? Then we could just rely on `last_consented_at` only and have "cleaner" data?

  - If so, I can write a backfill task and add it to this PR

Reply:
I'm liking the way you coded it.  I don't think we should do the backfill, etc.